src/routes/posts.js

```diff
@@ -1,13 +1,25 @@
-const express = require('express');
-const router = require('express').Router();
-const multer = require('multer');
-const { Post } = require('../handlers/Post');
+const express= require('express');
+const router= require('express').Router();
+const {Post}=require('../handlers/Post');
 
-const handler = new Post();
+const handler= new Post();
+router.get('/', handler.getAllPosts);
 
-const storage = multer.memoryStorage(); // Store files in memory
-const upload = multer({ storage: storage });
+router.delete('/:id', handler.deletePostById);
+router.post('/:id', handler.createPost);
```

Comment:
why do we need an id parameter while creating a post???

Reply:
i set it